setup.sh

```diff
@@ -37,17 +37,17 @@ if [[ -n $JAX_VERSION && ! ($MODE == "stable" || -z $MODE) ]]; then
      exit 1
 fi
 
-libtpu_path="/lib/libtpu.so"
+libtpu_path="$HOME/custom_libtpu/libtpu.so"
 
 if [[ "$MODE" == "libtpu-only" ]]; then
     # Only update custom libtpu.
     if [[ -n "$LIBTPU_GCS_PATH" ]]; then
         # Install custom libtpu
         echo "Installing libtpu.so from $LIBTPU_GCS_PATH to $libtpu_path"
         # Install required dependency
-        sudo pip3 install -U crcmod
+        pip3 install -U crcmod
         # Copy libtpu.so from GCS path
-        sudo gsutil cp "$LIBTPU_GCS_PATH" "$libtpu_path"
+        gsutil cp "$LIBTPU_GCS_PATH" "$libtpu_path"
```

Comment:
Do you think we should set `TPU_LIBRARY_PATH` in this script, whenever user provides `LIBTPU_GCS_PATH`. This way we can avoid user making the \$HOME vs. $HOME mistake. 
WDYT ?

Reply:
Good idea, but I don't think it help too much since `TPU_LIBRARY_PATH` will be reset each time you SSH into the TPU VM.